views/layouts/application.ejs

```diff
@@ -17,8 +17,8 @@
         crossorigin="anonymous" />
     <link href="https://fonts.googleapis.com/css?family=Montserrat:500|Open+Sans:400,400i,600|Fira+Sans&display=swap"
         rel="stylesheet" />
-    <link rel="stylesheet" type="text/css" href="/stylesheets/style.css" />
-    <link rel="stylesheet" type="text/css" href="/stylesheets/dark.css" media="(prefers-color-scheme:dark)" />
+    <link rel="stylesheet" type="text/css" href="/stylesheets/style.css?1" />
+    <link rel="stylesheet" type="text/css" href="/stylesheets/dark.css?1" media="(prefers-color-scheme:dark)" />
```

Comment:
Broke this selector:

https://github.com/mousetail/openletter/blob/1986da55ab58ca8841c53e636a8094d6d7a9229e/public/javascripts/application.js#L28

Should now use `*=` instead of `$=`.

Reply:
Oops, thanks for noticing